packages/blockchain-wallet-v4-frontend/src/components/Card/types.ts

```diff
@@ -0,0 +1,7 @@
+import { FC } from 'react'
+
+export type CardProps = {
+  elevation?: number
```

Comment:
Does it have a limit?

Reply:
not really, currently `elevation` is basically just being used as a boolean.

Reply:
but with the option to make the box-shadow elevate the box more or less in the future